css/custom.css

```diff
@@ -4168,6 +4168,16 @@ p.hangouts-social {
   background-position: -176px -238px;
 }
 
+.social-icon.discord {
+  background-image: url(../img/monero-spritesheet.png);
+  background-position: -128px -286px;
```

Comment:
`-129px -284px` looks better positioned than current one imo.

Reply:
Why they inconsistent?

some are
128 & 176
Github is 
127 & 177

some are increments of 48, some 47 or 46
facebook goes backward to 0 and -1

ofrn confused about inconsistencies

why arent they all:
128 + integer of 48
176 + integer of 48

Reply:
I'm more confused about why standalone icons aren't used.

Reply:
if you upload our spritesheet to https://getspritexy.netlify.app/ , it will make a little more sense

Reply:
#2314 corrects the coordinates for the existing icons, and notes that telegram is 45x45 instead of 43x43. And this PR adds a 45x46 discord icon

| Icon  | Default XY | Hover XY | Size
| --- | --- | --- | --- |
| Discord | 127 284 | 176 284 | 45x45 |